src/tools/sssctl/sssctl_user_checks.c

```diff
@@ -261,7 +261,8 @@ errno_t sssctl_user_checks(struct sss_cmdline *cmdline,
     ret = pam_start(service, user, &conv, &pamh);
     if (ret != PAM_SUCCESS) {
         ERROR("pam_start failed: %s\n", pam_strerror(pamh, ret));
-        return 1;
+        ret = 1;
```

Comment:
Since the function is returning an `errno_t`, we could change this `1` to `EPERM`. Don't you think?

Reply:
Fixed